src/js/components/DataTable/__tests__/__snapshots__/DataTable-test.tsx.snap

```diff
@@ -78,8 +78,8 @@ exports[`DataTable !primaryKey 1`] = `
   border-bottom: solid 1px rgba(0, 0, 0, 0.12);
   padding-left: 12px;
   padding-right: 12px;
-  padding-top: 6px;
-  padding-bottom: 6px;
+  padding-top: 12px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below